File: app/src/main/java/com/huck/biblequiz/data/local/entity/VerseEntity.kt
package com.huck.biblequiz.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "verses")
data class VerseEntity(
    @PrimaryKey val pk: Int,
    val bookId: Int,
    val chapter: Int,
    val verse: Int,
    val text: String
)

File: ios/BeSpecific/Data/Local/VerseEntity.swift
import Foundation
import SwiftData

@Model
final class VerseEntity {
    @Attribute(.unique) var pk: Int
    var bookId: Int
    var chapter: Int
    var verse: Int
    var text: String

    init(pk: Int, bookId: Int, chapter: Int, verse: Int, text: String) {
        self.pk = pk
        self.bookId = bookId
        self.chapter = chapter
        self.verse = verse
        self.text = text
    }
}
